vulture/core.py

```diff
@@ -130,6 +130,14 @@ def get_report(self, add_size=False):
             utils.format_path(self.filename), self.first_lineno,
             self.message, self.confidence, size_report)
 
+    def get_whitelist_string(self):
+        if self.typ != 'unreachable_code':
+            prefix = '_.' if self.typ in ['attribute', 'property'] else ''
+            return "{}{}  # unused {} ({}:{:d})".format(
+                    prefix, self.name, self.typ,
+                    utils.format_path(self.filename), self.first_lineno)
+        return ''
```

Comment:
Let's return a comment for unreachable code: `return '# {message} in file {filename} at line {first_lineno}'.format(**self.__dict__)`